openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -140,15 +139,12 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Mapping of Nova-managed servers to tags for current instance name
         self.external_host_tags = {}
 
-    def delete_apis_cache(self, instance_name):
-        for name, scope in list(iteritems(self._apis)):
-            if name == instance_name:
-                self.log.debug("Deleting instance api for instance: %s", name)
-                del self._apis[name]
+    def delete_api_cache(self):
+        del self._api
```

Comment:
Would setting this to `None` be safer, what do you think?

Reply:
Yes you're right, I will change it.